esphome/components/atm90e32/atm90e32.cpp

```diff
@@ -467,29 +440,426 @@ float ATM90E32Component::get_chip_temperature_() {
   return (float) ctemp;
 }
 
-uint16_t ATM90E32Component::calibrate_voltage_offset_phase(uint8_t phase) {
+void ATM90E32Component::run_gain_calibrations() {
+  if (!this->enable_gain_calibration_) {
+    ESP_LOGW("CALIBRATION", "Gain calibration is disabled! Enable it first with enable_gain_calibration: true");
```

Comment:
Logs should always use the TAG please. If you want to prefix with something then you just need to add it to the beginning of the message string. 

If you do want a different TAG for the calibration stuff, you can move these functions into their own c++ file with its own TAG eg `atm90e32_calibration.cpp` with `... TAG = "atm90e32.calibration"`

```suggestion
    ESP_LOGW(TAG, "[CALIBRATION] Gain calibration is disabled! Enable it first with enable_gain_calibration: true");
```